docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -47,17 +48,16 @@ Zowe Version 2.11.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
-
-#### DB2 Plug-in for Zowe CLI
+- Fixed an issue in the Daemon server which prevents users on Windows with uppercase letters in their username from using the Daemon.
```

Comment:
@adam-wolfe Is there an issue/PR we can add as a link here?

Reply:
https://github.com/zowe/zowe-cli/pull/1765